src/main/java/edu/ucla/library/iiif/auth/delegate/CantaloupeAuthDelegate.java

```diff
@@ -1,103 +1,132 @@
+import info.freelibrary.iiif.presentation.v3.services.auth.AuthCookieService1.Profile;
+import info.freelibrary.iiif.presentation.v3.services.auth.AuthTokenService1;
+import info.freelibrary.iiif.presentation.v3.utils.JSON;
+import info.freelibrary.iiif.presentation.v3.utils.JsonKeys;
 
-import edu.illinois.library.cantaloupe.delegate.AbstractJavaDelegate;
 import edu.illinois.library.cantaloupe.delegate.JavaDelegate;
 import edu.illinois.library.cantaloupe.delegate.Logger;
 
 /**
  * A Cantaloupe delegate for handing IIIF Auth interactions.
  */
-public class CantaloupeAuthDelegate extends AbstractJavaDelegate implements JavaDelegate {
+public class CantaloupeAuthDelegate extends GenericAuthDelegate implements JavaDelegate {
 
-    @Override
-    public String serializeMetaIdentifier(final Map<String, Object> aMetaIdentifier) {
-        return null;
-    }
+    /**
+     * A Jackson TypeReference for a Map.
+     */
+    private static final TypeReference<Map<String, Object>> MAP_TYPE_REFERENCE = new TypeReference<>() {};
 
-    @Override
-    public Map<String, Object> deserializeMetaIdentifier(final String aMetaIdentifier) {
-        return null;
+    /**
+     * A delegate configuration.
+     */
+    private final Config myConfig;
+
+    /**
+     * Creates a new Cantaloupe authorization delegate.
+     */
+    public CantaloupeAuthDelegate() {
+        myConfig = new Config();
     }
 
+    /**
+     * Authorizes a request for image information. Not all image information will necessarily be calculated at this
+     * point in time.
+     */
     @Override
     public Object preAuthorize() {
         Logger.info("The pre-authorize identifier is: " + getContext().getIdentifier());
         return true;
     }
 
+    /**
+     * Authorizes a request for an image.
+     */
     @Override
     public Object authorize() {
+        Logger.debug("authorize");
         return true;
     }
 
     @Override
     public Map<String, Object> getExtraIIIF2InformationResponseKeys() {
-        return Collections.emptyMap();
+        return getExtraInformationResponseKeys();
     }
 
     @Override
     public Map<String, Object> getExtraIIIF3InformationResponseKeys() {
-        return Collections.emptyMap();
-    }
-
-    @Override
-    public String getSource() {
-        return null;
-    }
-
-    @Override
-    public String getAzureStorageSourceBlobKey() {
-        return null;
-    }
-
-    @Override
-    public String getFilesystemSourcePathname() {
-        return null;
-    }
-
-    @Override
-    public Map<String, Object> getHTTPSourceResourceInfo() {
-        return null;
-    }
-
-    @Override
-    public String getJDBCSourceDatabaseIdentifier() {
-        return null;
-    }
-
-    @Override
-    public String getJDBCSourceMediaType() {
-        return null;
-    }
-
-    @Override
-    public String getJDBCSourceLookupSQL() {
-        return null;
+        return getExtraInformationResponseKeys();
```

Comment:
In lieu of changing `getAuthServices` now to conform to Image API 3, I think it would be acceptable to just leave this method unimplemented for now.

Reply:
Ick, not a great preview, GitHub. This method == `CantaloupeAuthDelegate::getExtraIIIF3InformationResponseKeys`, that is

Reply:
yes

Reply:
Because we're not actively using v3 yet or some other reason?